components/TodoList.tsx

```diff
@@ -1,3 +1,32 @@
-export function TodoList() {
-  return <></>;
+import { Text, ListItem, Stack } from "tamagui";
+
+import { TodoStore } from "../stores/Todo.store";
+import { observer } from "mobx-react-lite";
+
+type Props = { store: TodoStore };
+
+function EmptyTodo() {
+  return (
+    <Text>No to-dos created... make one with the button in the corner!</Text>
+  );
 }
+
+const ListTodos = observer(function ListTodos({ store }: Props) {
+  return store.todos.map((todo) => <ListItem title={todo.text} />);
+});
+
+const TodoContents = observer(function TodoContents({ store }: Props) {
+  if (store.todos.length === 0) {
+    return <EmptyTodo />;
+  } else {
+    return <ListTodos store={store} />;
+  }
+});
+
+export const TodoList = observer(function TodoList({ store }: Props) {
```

Comment:
As you can see now. You have prop drilling. So effectively the Store prop is defined in the root `App` component and it is passed down from one component to the other, all the way from ToDoList down to ListTodos.

This is something that we want to avoid in React.
You can use the React's Context, and provide the 'store' value via the provider and useContext hook.

useContext can be used by any component that needs to access the `store` instance.

Reply:
Okay, I'll port to using react context.